stock_dashboard_api/models/dashboard_config_models.py

```diff
@@ -0,0 +1,56 @@
+from stock_dashboard_api.utils import pool as db
+
+
+class DashboardConfig:
+    """
+    Model used to create a dashboard_config_hash.
+    """
+    _table = 'public.dashboard_config'
+
+    def __init__(self, config_hash: str, pk=None):
+        self.pk = pk
+        self.config_hash = config_hash
+
+    @classmethod
+    def create(cls, config_hash: str):
+        """
+        Creates a new config_hash.
+        """
+        with db.Connection() as conn:
+            query = f"""INSERT INTO {cls._table} (config_hash)
+                        VALUES (%(config_hash)s)
+                        RETURNING id, config_hash;"""
+            conn.cursor.execute(query, {'config_hash': config_hash})
+            pk, config_hash = conn.cursor.fetchone()
+            return DashboardConfig(pk=pk, config_hash=config_hash)
+
+    def update(self, config_hash=None):
```

Comment:
`self.config_hash` is not updated.
Also, there is no reason for config_hash to have None as default value, empty call is just meaningless